.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    # Check for updates once a day
```

Comment:
Can we do once every 2 weeks?

Reply:
Unfortunately dependabot doesn't support that.  Would you prefer weekly or monthly?

https://github.com/dependabot/dependabot-core/issues/2038

Reply:
Let's do weekly.